code/modules/mob/living/carbon/xenomorph/castes/hivelord/abilities_hivelord.dm

```diff
@@ -167,30 +167,36 @@ GLOBAL_LIST_INIT(thickenable_resin, typecacheof(list(
 	X.visible_message("<span class='xenonotice'>\The [X] digs out a tunnel entrance.</span>", \
 	"<span class='xenonotice'>We dig out a tunnel, connecting it to our network.</span>", null, 5)
 	var/obj/structure/tunnel/newt = new(T)
+
 	playsound(T, 'sound/weapons/pierce.ogg', 25, 1)
 
 
 	newt.creator = X
 
-
 	X.tunnels.Add(newt)
 
 	add_cooldown()
 
-	to_chat(X, "<span class='xenonotice'>We dig our tunnel to the nearest tunnel intersection, connecting it to the network. We now have [LAZYLEN(X.tunnels)] of [HIVELORD_TUNNEL_SET_LIMIT] tunnel sets.</span>")
+	newt.hud_set_xeno_tunnel()
+
+	to_chat(X, "<span class='xenonotice'>We now have <b>[LAZYLEN(X.tunnels)] of [HIVELORD_TUNNEL_SET_LIMIT]</b> tunnels.</span>")
 
-	var/msg = stripped_input(X, "Add a description to the tunnel:", "Tunnel Description")
-	newt.tunnel_desc = "[get_area(newt)] (X: [newt.x], Y: [newt.y]) [msg]"
-	newt.name += " [msg]"
+	var/msg = stripped_input(X, "Give your tunnel a descriptive name:", "Tunnel Name")
+	newt.tunnel_desc = sanitize("[get_area(newt)] (X: [newt.x], Y: [newt.y])")
+	newt.name += sanitize(" [msg]")
```

Comment:
Stripped input handles sanitizing already both of these are unneeded

Reply:
Done.